docs/tutorials/index.md

```diff
@@ -0,0 +1,186 @@
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+* This will be replaced by the TOC
+{:toc}
+
+## Goals and Scope of these Tutorials
+
+These tutorials present an introduction to Apache Flink that includes just enough to get you started
+writing scalable streaming ETL, analytics, and event-driven applications, while leaving out a lot of
+(ultimately important) details. The focus is on providing straightforward introductions to Flink's
+APIs for managing state and time, with the expectation that having mastered these fundamentals,
+you'll be much better equipped to pick up the rest of what you need to know from the more detailed
+reference documentation. The links at the end of each page will lead you to where you can learn
+more.
+
+Specifically, you will learn:
+
+- how to implement streaming data processing pipelines
+- how and why Flink manages state
+- how to use event time to consistently compute accurate analytics
+- how to build event-driven applications on continuous streams
+- how Flink is able to provide fault-tolerant, stateful stream processing with exactly-once semantics
+
+These tutorials focus on four critical concepts: continuous processing of streaming data, event
+time, stateful stream processing, and state snapshots. This page introduces these concepts.
+
+{% info Note %} Accompanying these tutorials are a set of hands-on exercises that will guide you
+through learning how to work with the concepts being presented.
+
+{% top %}
+
+## Stream Processing
+
+Streams are data's natural habitat. Whether it's events from web servers, trades from a stock
+exchange, or sensor readings from a machine on a factory floor, data is created as part of a stream.
+But when you analyze data, you can either organize your processing around _bounded_ or _unbounded_
+streams, and which of these paradigms you choose has profound consequences.
+
+<img src="{{ site.baseurl }}/fig/bounded-unbounded.png" alt="Bounded and unbounded streams" class="offset" width="90%" />
+
+**Batch processing** is the paradigm at work when you process a bounded data stream. In this mode of
+operation you can choose to ingest the entire dataset before producing any results, which means that
+it's possible, for example, to sort the data, compute global statistics, or produce a final report
+that summarizes all of the input.
+
+**Stream processing**, on the other hand, involves unbounded data streams. Conceptually, at least,
+the input may never end, and so you are forced to continuously process the data as it arrives. 
+
+In Flink, applications are composed of **streaming dataflows** that may be transformed by
+user-defined **operators**. These dataflows form directed graphs that start with one or more
+**sources**, and end in one or more **sinks**.
```

Comment:
You could actually link to the glossary here, but I'm not sure whether it would confuse people if they actually followed the links and read the information there (may be too much detail there); however, it may be useful later.

Reply:
I'm trying to minimize links within each page, to encourage folks to continue reading until the end. And yes, I think going to the glossary this early would be a mistake.